amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseTakerIntegrationTest.java

```diff
@@ -105,7 +104,7 @@ public void testNonGreedyTake() throws LeasingException {
 
         builder.withLease("4", "bar").build();
 
-        builder.takeMutateAssert(taker, 2);
+        builder.takeMutateAssert(taker, 3);
```

Comment:
Hmmm, we need the test case for new and recently renewed leases as well. Here the workers should be taking 2 each.